identity-service/prod.env

```diff
@@ -0,0 +1,37 @@
+dbUrl=postgres://postgres:postgres@db:5432/postgres
+
+redisHost=cache
+redisPort=6379
+
+aaoAddress=0x9811BA3eAB1F2Cd9A2dFeDB19e8c2a69729DC8b6
+aaoEndpoint=https://antiabuseoracle.audius.co
+amplitudeAPIKey=86760558b8bb1b3aae61656efd4ddacb
+blacklisterPublicKey=0xfeebEA99dE524ac668B6f151177EcA60b30A09c9
+cognitoBaseUrl=https://api.cognitohq.com
+cognitoTemplateId=flwtmp_eMMe8VQGrKvLCX
+ethBridgeAddress=0x98f3c9e6E3fAce36bAAd05FE09d375Ef1464288B
+ethOwnerWallet=0x3eC41B55D9A1Edd593Ef38493d8125DeB4EE03a9
+ethRegistryAddress=0xa0fd5C42F81CC35aa66E5f8Ee85098fF7D669fa6
+ethTokenAddress=0x2999e02829DC711B9254187962ba44d1fFcf5481
+ethTokenBridgeAddress=0x3ee18B2214AFF97000D974cf647E7C347E8fa585
+minimumBalance=25
+registryAddress=0x65D2F689a0ea3A5D1Ee97f7F8f3a9549860b9Ed5
+relayerPublicKey=0xdead88167Bd06Cbc251FB8336B44259c6407dd07
+rewardsParallelization=10
+rewardsQuorumSize=3
+secondaryWeb3Provider=https://core.poa.network
+solBridgeAddress=worm2ZoG2kUd4vFXhvjh93UUH596ayRfgQ2MgjNMTth
+solTokenBridgeAddress=wormDTUJ6AWPNvk59vGQbDvGJmqbDTdgWgAqcLBCgUb
+solanaClaimableTokenProgramAddress=Ewkv3JahEFRKkcJmpoKB7pXbnUHwjAyXiwEo4ZY2rezQ
+solanaMintAddress=9LzCMqDgTKYz9Drzqnpgee3SGa89up3a247ypMj2xrqM
+solanaRewardsManagerProgramId=DDZDcYdQFEMwcu2Mwo75yGFjJ1mUQyyXLWzhZLEVFcei
+solanaRewardsManagerProgramPDA=71hWFVYokLaN1PNYzTAWi13EfJ7Xt9VbSWUKsXUT8mxE
+solanaRewardsManagerTokenPDA=3V9opXNpHmPPymKeq7CYD8wWMH8wzFXmqEkNdzfsZhYq
+userVerifierPublicKey=0xbeef8E42e8B5964fDD2b7ca8efA0d9aef38AA996
+web3Provider=https://poa-gateway.audius.co
+wormholeRPCHosts=https://wormhole-v2-mainnet-api.certus.one,https://wormhole.inotel.ro,https://wormhole-v2-mainnet-api.mcf.rocks,https://wormhole-v2-mainnet-api.chainlayer.network,https://wormhole-v2-mainnet-api.staking.fund,https://wormhole-v2-mainnet-api.chainlayer.network
+
+ethProviderUrl=
```

Comment:
is the plan to leave these empty and make consumers required to fill this out in the override?

Reply:
Yes, that's right